src/python/zquantum/core/utils.py

```diff
@@ -536,11 +539,39 @@ def load_nmeas_estimate(filename: str) -> Tuple[float, int, np.ndarray]:
         frame_meas: frame measurements (number of measurements per group)
     """
 
-    with open(filename, 'r') as f:
+    with open(filename, "r") as f:
         data = json.load(f)
 
-    frame_meas = convert_dict_to_array(data['frame_meas'])
-    K_coeff = data['K']
-    nterms = data['nterms']
+    frame_meas = convert_dict_to_array(data["frame_meas"])
+    K_coeff = data["K"]
+    nterms = data["nterms"]
 
     return K_coeff, nterms, frame_meas
+
+
+def hf_rdm(n_alpha: int, n_beta: int, n_orbitals: int):
```

Comment:
Can provide type hinting for the return value with `-> InteractionRDM`

Reply:
Done now, and also added a test for `hf_rdm` (although it requires to store Hamiltonians which is quite ugly, if there's a better way please let me know)

Reply:
Best thing to do for large dicts is to put them in a separate file. This change is in aa99939d1922d0773f417c6815c88f402a1f2ddb